Sources/WordPressKitModels/RemoteUser.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+@objcMembers public class RemoteUser: NSObject {
+    public var userID: NSNumber?
+    public var username: String?
+    public var email: String?
+    public var displayName: String?
+    public var primaryBlogID: NSNumber?
+    public var avatarURL: String?
+    public var dateCreated: Date?
+    public var emailVerified: Bool = false
+    public var linkedUserID: NSNumber?
```

Comment:
The original properties in the original Objective-C code are `null_unspecified` or implicitly unwrapped optional. I can keep the original code and make the new properties IUO, too. But that does not _feel_ right.

The new code now uses optional, which means the app will need to update their codebase to deal with the new optional properties instead of IUO properties.